README.md

```diff
@@ -444,6 +445,24 @@ Note: For modules use the [@import](http://clang.llvm.org/docs/Modules.html#usin
 #import "NYTUserView.h"
 ```
 
+## Protocols
+
+In a delegate protocol, the first parameter to each method should be the object sending the message.
```

Comment:
hmm, makes sense. But now that you suggest that, is there some more general underlying principle at hand? Is there a better term to describe "delegate, data source, other protocols which exist for tight collaboration between objects"?